docs/concepts/ipfs-implementations.md

```diff
@@ -16,29 +19,112 @@ Looking for an easy, user-friendly way to get started with IPFS? Try any of the
 :::
 
-## Popular or Actively Maintained
+The implementation or tool that you use to interact with and use IPFS depends on your particular use case. 
+
+- [Use IPFS with your browser](#use-ipfs-with-your-browser)
+- [Use IPFS without code](#use-ipfs-without-code)
+- [Interact via the CLI](#interact-via-the-cli)
+- [Develop applications](#develop-applications)
+
+### Use IPFS with your browser
+
+Interact with the network using any Brave, Chrome, Edge, Firefox, Opera, and any other Chromium-based web browser using the [IPFS Companion browser extension](../install/ipfs-companion.md).
+
+### Use IPFS without code
+
+- If you'd like desktop-native GUI application, IPFS Desktop bundles an IPFS node, file manager, peer manager, and content explorer together. No need to touch the terminal or use code. Just [install Desktop](../install/ipfs-desktop.md) and quickly access the network.
+- To store data on IPFS persistently, without dealing with any applications or installations, you can also use a [pinning service](../concepts/persistence.md#pinning-services). You can follow [this quickstart guide](../quickstart/publish.md) to learn more.
+
+### Interact via the CLI
+
+You can interact with the IPFS network via the terminal:
+
+- To quickly retrieve data with minimal complexity and overhead, use the [Lassie client](https://github.com/filecoin-project/lassie/), which provides a [simple command-line interface](https://github.com/filecoin-project/lassie/#command-line-interface), in addition to other features.
+- For a wider range of features, use [IPFS Kubo as a command-line tool](../reference/kubo/cli.md).
+
+### Develop applications
+
+You can develop IPFS applications with native support in languages such as [Go](#go) or [JavaScript](#javascript), or by [using HTTP](#http). There are also tools available to support [specialized use cases such as data orchestration and mobile applications](#specialized-use-cases).
+
+#### HTTP
+
+If you'd like to use IPFS over standard HTTP, you have several options:
+
+- If you simply want to retrieve data, use the [Lassie client HTTP API](https://github.com/filecoin-project/lassie/#http-api).
+- You can control an IPFS Kubo node using HTTP via the [Kubo RPC API](../reference/kubo/rpc.md), with the same range of functionality available via the [Kubo CLI](kubo/cli.md). There are multiple [RPC API clients in multiple languages](../reference/kubo-rpc-cli.md) to choose from.
+- For an implementation and runtime agnostic HTTP interface, use an [IPFS gateway](../reference/http/gateway.md).
+
+#### Go
+
+Looking for to add IPFS functionality to your Go application? Try these implementations:
+
+- To quickly retrieve data with minimal complexity and overhead, use the [Lassie client Golang library](https://github.com/filecoin-project/lassie/#golang-library) in your applications.
+- For more complicated applications requiring the full range of IPFS functionality, use Kubo or Boxo.
```

Comment:
I think we want to point people at boxo, then kubo, then the rest.
(Lassie is limited to "leach" mode, it should not be suggested as the first option)

```suggestion
- To create custom IPFS node tailored to your individual needs, try [Boxo SDK](https://github.com/ipfs/boxo).
- For applications requiring the full range of IPFS functionality, such as  providing data to other peers, use [Kubo](https://github.com/ipfs/kubo).
- To quickly retrieve data with minimal complexity and overhead, use the [Lassie client Golang library](https://github.com/filecoin-project/lassie/#golang-library) in your applications.
```